pandas/io/sql.py

```diff
@@ -11,6 +11,7 @@
 import warnings
 
 import numpy as np
+from sqlalchemy import exc
```

Comment:
Question: I need `exc` to catch the exception but is this the right place to have this import? I notice that sqlalchemy itself is imported by helper function - is that where this should go?

Reply:
This will need to be a runtime import i.e. put it near where you are catching the exception